.md

```diff
@@ -0,0 +1,40 @@
+.frame{
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 236px;
+  height: 32px;
+
+  /* Inside auto layout */
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.tag {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 8px;
+  width: 214px;
+  height: 24px;
```

Comment:
- [ ] I think it's better not to give any HTML element a fixed ``width`` and ``height`` as I think this is one of the main reasons that cause your design to not work the way you want, but rather use ``margin`` and ``padding`` to size your elements in a way that is **responsive** [Check this video](https://www.youtube.com/watch?v=N7zPL8i_DTs)